alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md

```diff
@@ -1,39 +1,4 @@
-# KubeVirtComponentExceedsRequestedCPU
+# KubeVirtComponentExceedsRequestedCPU [Deprecated]
 
-## Meaning
-
-This alert fires when a component's CPU usage exceeds the requested limit.
-
-## Impact
-
-Usage of CPU resources is not optimal and the node might be overloaded.
-
-## Diagnosis
-
-1. Set the `NAMESPACE` environment variable:
-
-   ```bash
-   $ export NAMESPACE="$(oc get kubevirt -A \
-     -o custom-columns="":.metadata.namespace)"
-   ```
-
-2. Check the component's CPU request limit:
-
-   ```bash
-   $ oc -n $NAMESPACE get deployment <component> -o yaml | grep requests: -A 2
-   ```
-
-3. Check the actual CPU usage by using a PromQL query:
-
-   ```text
-   node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate
-   {namespace="$NAMESPACE",container="<component>"}
-   ```
-
-See the
-[Prometheus documentation](https://prometheus.io/docs/prometheus/latest/querying/basics/)
-for more information.
-
-## Mitigation
-
-Update the CPU request limit in the `HCO` custom resource.
\ No newline at end of file
+This alert has been deprecated; it does not indicate a genuine issue.
```

Comment:
if you want this to be splitted into two lines at the rendered markdown page, you should add two spaces at the end of the first line.
```suggestion
This alert has been deprecated; it does not indicate a genuine issue.  
```

Reply:
are you sure? I tested it in both states (with and without the space and same results), see https://github.com/openshift/runbooks/blob/8434876c33a38463387320f3880336ffc4c30068/alerts/openshift-virtualization-operator/KubeVirtComponentExceedsRequestedCPU.md